Monika After Story/game/script-topics.rpy

```diff
@@ -15985,27 +15985,34 @@ label monika_eating_meat:
-    m 2etc "At that point, is it really a better alternative as far as the environment is concerned? {w=0.3}{nw}"
-    extend 4euc "Not to mention, I still want to have a varied and balanced diet."
-    m 4eud "Vegan diets can be quite deficient in nutrients, such as vitamin B12, calcium, iron, and zinc."
+    m 2etc "Then again, these are obviously not vegan exclusive issues, but the exclusion of more food groups naturally brings the consumption of these products up."
+    m 1rud "So we'll have to carefully weigh the pros and cons of each food group we include, and research on vegan diets is kind of new compared to vegetarian diets."
+    m 4eua "Not to mention, I still want to have a varied diet with all the flavors that I like."
+    m 4eud "Vegan diets can be somewhat deficient in essential nutrients if not planned properly, such as vitamin B12, calcium, iron, and zinc."
     m "Granted, there are still some options including supplements, but balancing a vegan diet takes a lot of care and thought."
-    m 7eka "...So for that reason, I'm not personally against eating things like milk and eggs. {w=0.2}But I think I'd prefer to buy locally if possible."
+    m 7eka "...So for that reason, I'm not personally against consuming things like milk and eggs on occasion, even if it does have a slightly higher environmental impact."
+    m "But I think I'd prefer to buy locally if possible."
     m 3eud "Farmer's markets are great places to buy food, {w=0.2}even meat, {w=0.2}produced with less of an environmental impact."
     m 3ekd "But they can typically be pretty expensive...and depending on location, leave you with fewer options. {w=0.3}{nw}"
     extend 3eua "So I'm okay with buying from a plain old store, if need be."
     m 1euc "As for meat that comes from local hunting and fishing, I think that's alright to eat as well, but it's important to research what areas might be over-hunted, and what animals to be careful of."
     m 3rtc "That said, I don't know that I'd {i}prefer{/i} to eat meat, given the option."
     m 3eka "Since I've adjusted myself to a vegetarian diet, my palate has changed to prefer certain flavors."
-    m 3rksdla "I don't hate the taste of meat, but I don't think I'd want it to be a major part of my meal, either."
-    m 1eka "...But if you prepared something with meat, I could try a little bit as a side dish... {w=0.3}{nw}"
-    extend 3hub "That way I can still enjoy your cooking!"
+    m 3rksdla "I barely remember the taste of meat at this point, so it's hard to say if I would enjoy it very much."
+    m 1eka "But that's probably not going to be an issue by the point I cross over..."
+    m 1eub "Especially since there are a lot of good substitues for meat in supermarkets already, with way less environmental impact."
+    m 3rusdlc "It's hard to think about us ever ending up in a situation where we had to hunt for our survival."
```

Comment:
I'm not sure how much I like this part.

Her being open to trying it I think made sense. To me, she'd be open to having a little bit on some rare occasions, but definitely not a normal thing. I don't mind the dialogue you have here, but I think we can add that original notion of her being open to trying/having it on some occasions back.

Reply:
In the beginning of the topic, she already says she would do so for survival reasons. I think it's a bit too much to stretch it to the point where she would try some "just because" after that. It kind of doubles down on the pro-meat side instead of reiterating her commitment to staying vegetarian for the environment. When I wrote the re-evaluation of the vegan side of things, I made sure to not have her say anything indicating she would ever try cutting out milk or eggs or anything like that from her diet. Because as much as that would make me happy personally, she is a vegetarian, and I believe it's important to respect that when writing topics for her.

Reply:
> It kind of doubles down on the pro-meat side instead of reiterating her commitment to staying vegetarian for the environment

The line we have right now is neutral. It doesn't enforce she'll eat meat every time you're cooking, it's not "just because" either.

Reply:
Well, it's not neutral, it's more positive to eating meat than anything. This is not about me trying to say she would never eat meat. The rest of the topic covers scenarios where she would (survival situations). I just think it's unnecessary to keep a line that basically means she wants to postpone her convictions of not contributing to environmental degradation without any basis for that from the previous topic. It's shifting her position a little bit more than necessary. If you look at how I wrote about veganism, I made a lead-in focused on the people who are vegan's view on things, followed by her personal opinion, leading into a pros and cons session where she concludes with her own position again. None of that suggests she yields to the proposition, unlike these lines where she basically says she'd put aside her own convictions for a night or two.

Reply:
See, if she was a vegan, I could agree with that that line doesn't work, but she isn't. I can see her being open to try a bit of a meat dish her partner prepped.
"I could try a little bit as a side dish" - doesn't enforce she'd eat it on a daily basis, and doesn't shift her position. You changed `I don't hate the taste of meat` into `it's hard to say if I would enjoy it very much`. Which made things sound more negative. These changes are unnecessary. The only thing I like is that you mentioned meat substitutes here.

Reply:
I researched vegetarian positions on eating meat for rewriting this, and let me tell you, you would be hard pressed to find many vegetarians that would be open to "trying meat" as a side dish, even if their spouse made them that for dinner. It kind of becomes a natural part of who you are. The way Monika talks about it in the first topic and the rest of this topic, suggests she'd not prefer it at all, and if given the option would choose something else. I know in many other areas of life, "indulging" in food you'd not normally consume is normal, like eating sugar or a bag of chips, even if you want to stay slim. But that's more of a personal choice, unlike the choice to reduce your carbon footprint, which affects other people (including you, the player).

Reply:
When originally writing this topic, I actually wasn't sure if I liked this ending bit very much. I wanted to emphasize an experience that a lot of vegetarians I know have gone through--which is that over time, your palate changes such that meat tastes become more overwhelming or unpleasant. A few of those friends have said that they're still tempted by the occasional meat or fish dish, but the taste is so different to them that it needs to be a smaller portion so as not to overwhelm them or, in the case of those who have been vegetarian for many years, even make them a bit physically sick. I'm not sure that I conveyed that as well as I'd like, since I thought this was something she'd probably phrase more gently.

There was a bit of disagreement on whether or not she'd still eat meat if it was ethically sourced, so I wrote that she'd try a little bit as a compromise. If I were to rewrite it I think I'd add more to emphasize that ethical sourcing would be a major point in the equation.